3.evaluate_model/README.md

```diff
@@ -17,6 +17,15 @@ The f1 score data are saved to [F1_scores](evaluations/F1_scores).
 In [class_PR_curves.ipynb](class_PR_curves.ipynb), we use [sklearn.metrics.precision_recall_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html) to derive the precision-recall curves for each model, feature type, and dataset combination.
 The precision recall curves and their data are saved to [class_precision_recall_curves](evaluations/class_precision_recall_curves/).
 
+In [get_LOIO_probabilities.ipynb](get_LOIO_probabilities.ipynb), we use the optimal hyperparameters from each final logistic regression model (DP, CP, CP_and_DP) to fit and evaluate new models in a Leave One Image Out (LOIO) fashion.
+The LOIO evaluation procedure is as follows:
+- Load in entire MitoCheck labeled cell dataset (from [labeled_data.csv.gz](../data/labeled_data.csv.gz))
+- For each image in MitoCheck labeled cell dataset (as specified by the `Metadata_DNA` field):
+    - Train a logistic regression model with optimal hyperparameters (`C` and `l1_ratio`, saved with model data in [models/](../2.train_model/models/)) on every cell that is **not** in the specific image.
+    - Predict probabilities on every cell that **is** in the specific image.
```

Comment:
Are these items sequential? If so, consider using a numbered or lettered list instead of an unordered/bulleted one.

Reply:
Used numbered list in https://github.com/WayScience/phenotypic_profiling_model/pull/23/commits/0ca85f5f0e452c6651eeb2d8d24464bb24058b62